packages/core-bookmarks/bookmarks.ts

```diff
@@ -21,14 +21,24 @@ export const services = {
 };
 
 export interface UnchainedBookmarkAPI {
-  findByUserId({ userId: String }): Promise<Array>;
+  findByUserId(userId: string): Promise<Array>;
+  findByUserIdAndProductId({ userId: string, productId: string }): Promise<any>;
```

Comment:
don't use `any`. Shoud probably be `Promise<Bookmark[]>` or `Promise<Bookmark>` as there can only be one